src/NuGet.Clients/NuGet.SolutionRestoreManager.Interop/NuGet.SolutionRestoreManager.Interop.csproj

```diff
@@ -43,13 +43,13 @@
   <Import Project="$(BuildCommonDirectory)common.targets" />
 
   <!-- Custom pack, called by build.proj -->
-  <Target Name="Pack" DependsOnTargets="Build">
+  <Target Name="Pack">
```

Comment:
@alpaix since this is not a netcore project, it can not be packing using dotnet.exe or msbuild.